data/map.txt

```diff
@@ -13648,8 +13651,11 @@ system "Hi Yahr"
 	trade Medical 825
 	trade Metal 308
 	trade Plastic 372
+	fleet "Unfettered Logistics" 600
```

Comment:
Lower numbers are higher spawns fam

Reply:
~~Well it needs to be lower then you got my poit-nt~~

Reply:
```suggestion
	fleet "Unfettered Logistics" 450
```
There we go, should be good with this
It shouldnt be too busy either with this amount since the regular fleets were cut in half
Lets just do this in this PR and I'll adjust it further for UHai in another pr if needed